R/PipeOpEnsemble.R

```diff
@@ -151,10 +166,11 @@ weighted_matrix_sum = function(matrices, weights) {
 # @return `factor`
 weighted_factor_mean = function(factors, weights, alllevels) {
```

Comment:
this used to return a `factor` with the levels as arg max with respect to the `prob` matrix and is used this way by `LearnerClassifAvg`: https://github.com/mlr-org/mlr3pipelines/blob/97f38fa14ece48ae887d7bd8e7e1a1d4d7b88386/tests/testthat/test_learner_weightedaverage.R#L12 Maybe we should add a flag to have a fallback to this return value? Or rather change `LearnerClassifAvg` directly?

Reply:
I think LearnerClassifAvg should be changed